lib/ttdrest/concerns/campaign.rb

```diff
@@ -83,6 +83,24 @@ def build_campaign_data(campaign_id, name, budget, start_date, campaign_conversi
           campaign_data = campaign_data.merge({"DailyBudgetInImpressions" => params[:daily_budget_in_impressions]})
         end
 
+        # Latest KOA fields
```

Comment:
does this only apply on create OR does it action on update as well?

Reply:
yeah TTDRest is... not super useful these days its just a "does whatever you tell it to" interface to the apis, but it will only honor/map attributes it's been told about